opentracing-api-extensions-tracer/src/test/java/io/opentracing/contrib/api/tracer/APIExtensionsFullSpanLifecycleTest.java

```diff
@@ -25,19 +25,6 @@
 
 public class APIExtensionsFullSpanLifecycleTest {
 
-    @Test
-    public void testMockTracerActiveSpan() {
-        APIExtensionsTracer extTracer = new APIExtensionsTracer(new MockTracer());
-        TracerObserver tracerObserver = Mockito.mock(TracerObserver.class);
-        SpanObserver spanObserver = Mockito.mock(SpanObserver.class);
-        Mockito.when(tracerObserver.onStart(Matchers.any(SpanData.class))).thenReturn(spanObserver);
-        extTracer.addTracerObserver(tracerObserver);
-
-        extTracer.buildSpan("testOp").startActive(true).close();
-        
-        Mockito.verify(spanObserver, Mockito.times(1)).onFinish(Matchers.any(SpanData.class), Matchers.anyLong());
-    }
-
     @Test
     public void testMockTracerManualSpan() {
```

Comment:
nit: as there is no distinction now between manual and active, could you change this test name to `testMockTracerSpan`?